### PR TITLE
feat(web/applications): add/ delete representations attachment

### PR DESCRIPTION
**Pull request by jonathangoulding**
## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

**Review by francescobrenci-AR — APPROVED** [review]

File: apps/api/src/server/applications/application/representations/attachment/attachment.service.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import * as representationsRepository from '../../../../repositories/representation.repository.js';
+
+export const addAttachmentRepresentation = async (repId, documentId) => {
```
Comment (Philip-Philippou1) [inline apps/api/src/server/applications/application/representations/attachment/attachment.service.js:3]: Small naming question is this intended as in - add attachment to representation ?

Fine if it is.

File: apps/api/src/server/applications/application/representations/attachment/attachment.route.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        #swagger.description = 'Add a representation attachment for an application document'
+        #swagger.parameters['id'] = {
+            in: 'path',
+			description: 'Application ID',
+			required: true,
+			type: 'integer'
+		},
+				#swagger.parameters['body'] = {
+            in: 'body',
+            description: 'Representation Details',
```
Comment (Philip-Philippou1) [inline apps/api/src/server/applications/application/representations/attachment/attachment.route.js:25]: Probably more 'Document details'

File: apps/api/src/server/applications/application/representations/attachment/attachment.route.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        #swagger.path = '/applications/{id}/representations/{repId}/attachment/{attachmentId}'
+        #swagger.description = 'Delete a representation attachment for an application document'
+        #swagger.parameters['id'] = {
+            in: 'path',
+			description: 'Application ID',
+			required: true,
+			type: 'integer'
+		},
+        #swagger.responses[200] = {
+            description: 'Representation',
```
Comment (Philip-Philippou1) [inline apps/api/src/server/applications/application/representations/attachment/attachment.route.js:56]: Probably description: attachment

File: apps/api/src/server/applications/application/representations/attachment/attachment.service.js (hunk 1 above)
Comment (jonathangoulding) [reply apps/api/src/server/applications/application/representations/attachment/attachment.service.js:3]: this follows the other naming conventions for the services.

**Review by Philip-Philippou1 — APPROVED** [review]

**Pull request merged**